### ISSUE TITLE
[Error] Service "payment.api" failed to build The command '/bin/sh -c dotnet restore -nowarn:msb3202,nu1503

### ISSUE THREAD

**Issue by ddraig611** [dotnet-architecture/eShopOnContainers#681]
Something, when building all solution "eShopOnContainers-ServicesAndWebApps.sln" 
will appear error "Service 'payment.api' failed to build: The command '/bin/sh -c dotnet restore -nowarn:msb3202,nu1503' returned a non-zero code: 1."
![image](https://user-images.githubusercontent.com/41619054/43239262-ba923204-90bb-11e8-921b-1a140f25e913.png)
this issue isn't occur on the ubuntu
Please help me resolve this issue
Can u help?

**Comment by mvelosop** [comment]
Hi, @ddraig611,

Please update your repo to the latest commit (e7ac8737669af446cfab6b0298460f3f6597acb7) and try again.

Also check that no containers are running before starting, i.e. `docker ps` should be blank.

If it fails, post the the output from the build process, this part: (but as text, not as image):
![image](https://user-images.githubusercontent.com/7357806/43315036-ac78be00-918c-11e8-88bb-a153acb805f1.png)

**Comment by mvelosop** [comment]
**UPDATE** - I just got to this error on another computer and it "seems" it was caused by **Payment.API** project being left out of **eShopOnContainers-ServicesAndWebApps.sln**.

I didn't quite understood what happened because the problem didn't show on the computer I usually work with but id did in another one, even though both where on the same commit and I removed all containers and images. Anyway the issue seems to be solved now,  as of commit: e1313f2c2ad814d789ccd19184f97f42b656a882.

Hope this helps.
